libbindgen/src/lib.rs

```diff
@@ -168,8 +168,8 @@ impl Builder {
     /// Set the input C/C++ header.
     pub fn header<T: Into<String>>(mut self, header: T) -> Builder {
         let header = header.into();
-        self.options.input_header = Some(header.clone());
-        self.clang_arg(header)
+        self.options.input_header = Some(header);
```

Comment:
That doesn't quite fit with the rest of the builder methods, I think overriding it, albeit uncommon, should be fine?